### ISSUE TITLE
Issue syncing boards.

### ISSUE THREAD

**Issue by thehalestorm** [avocode/avocode#1556]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.7`
Avocode UID: 148691
```
Error: ENOENT: no such file or directory, stat '/var/folders/64/05751jwj3db21rhqjtrd12_c0000gn/T/com.madebysource.avocode/avocode-sync-1501093078709/53FD267D-85C5-4E9E-A77C-69E68171967F/E088C708-177D-489B-82C0-F292000EA054.png'
```

**Comment by jakubzitny** [comment]
Hi @thehalestorm, thanks for reporting this. Can you please try to download [this version](https://upx.cz/iYd) of plugin, double click to update it in Sketch and try again?

**Comment by jakubzitny** [comment]
Hi, we've fixed this problem in latest Avocode `2.14.8`, please update the app and let us know if you have other problems or issues :wink:
